tests/resources/promise-rejection-events.js

```diff
@@ -446,14 +446,14 @@ async_test(function(t) {
    'all inside a setTimeout');
 
 //
-// Positive unhandledrejection/rejectionhandled tests with delayed attachment
+// Negative unhandledrejection/rejectionhandled tests with task-delayed attachment
 //
 
 async_test(function(t) {
   var e = new Error();
   var p;
 
-  onUnhandledSucceed(t, e, function() { return p; });
+  onUnhandledFail(t, function() { return p; });
```

Comment:
so this test fails. I think the test is wrong... 

the task to send the unonunhandled event is created in the microtask checkpoint. the task that attaches a handler then runs before, but at that point, the task for the onunhandled event already exists, and it won't be cancelled.